### PR TITLE
cgame: allow missile camera in demo playback

### PR DESCRIPTION
**Pull request by mittermichal**
useful for reviewing demos after game or making videos

### CODE REVIEW

File: src/cgame/cg_ents.c [hunk 1]
```diff
@@ -1432,7 +1432,7 @@ static void CG_Missile(centity_t *cent)
 		}
 	}
 
-	if (cgs.sv_cheats && cent->currentState.clientNum == cg.predictedPlayerState.clientNum)
+	if ((cgs.sv_cheats || !cg.demoPlayback) && cent->currentState.clientNum == cg.predictedPlayerState.clientNum)
```
Comment (Aciz) [inline src/cgame/cg_ents.c]: This does the exact opposite :D it should be like this

```suggestion
	if ((cgs.sv_cheats || cg.demoPlayback) && cent->currentState.clientNum == cg.predictedPlayerState.clientNum)
```

Comment (mittermichal) [reply src/cgame/cg_ents.c]: damn, I tested it though 😄

Comment (mittermichal) [reply src/cgame/cg_ents.c]: it seemed working because I had sv_cheats 1 😄

Comment (mittermichal) [reply src/cgame/cg_ents.c]: fixed

**Pull request merged**